datamodel/src/main/kotlin/com/hartwig/actin/datamodel/molecular/driver/VariantAlteration.kt

```diff
@@ -0,0 +1,5 @@
+package com.hartwig.actin.datamodel.molecular.driver
+
+interface VariantAlteration : GeneAlteration {
```

Comment:
I would make this a "data class" rather than an interface. A data class is typically meant for types where you make instances of (which you do), while an interface is usually meant to define to set of properties/functions that are common across different (data) classes.